client/js/constants.js

```diff
@@ -1,4 +1,5 @@
 "use strict";
+const modes = require("./libs/handlebars/modes");
```

Comment:
Can this be done with `import { modes } from ...`?

Reply:
Possibly, but do we allow es6 imports just now?

Reply:
I think we do.

Reply:
I believe we can since #1726, except in files that are shared between the client and the server :(